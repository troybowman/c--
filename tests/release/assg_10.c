../tree/assg_10.c