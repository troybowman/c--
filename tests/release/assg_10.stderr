error: use of undeclared identifier argv at line 1
syntax error, line: 1
c-- returned nonzero exit code